.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+    name: JDK ${{ matrix.java_version }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java_version: [1.8, 11]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Install JDK
```

Comment:
GitHub fixed a bug that prevented using replacements in names, so you could do `Install JDK ${{ matrix.java_version }}` here if you want. Haven't verified it works yet but someone else on the community forum confirmed it

Reply:
Oh cool, yeah, I'll do it!